src/moonstone-samples/lib/DataGridListSample.js

```diff
@@ -40,26 +83,28 @@ module.exports = kind({
 	pattern: 'activity',
 	classes: 'moon enyo-fit enyo-unselectable',
 	components: [
-		{kind: Panel, classes: 'moon-6h', title: 'Menu', components: [
-			{kind: Item, content: 'Scroll'},
-			{kind: Item, content: 'the'},
-			{kind: Item, content: 'Data Grid List'},
-			{kind: Item, content: 'to'},
-			{kind: Item, content: 'the'},
-			{kind: Item, content: 'Right!'}
-		]},
-		{kind: Panel, joinToPrev: true, title: 'Data Grid List', headerComponents: [
-			{kind: ToggleButton, content: 'Selection', name: 'selectionToggle'},
+		{kind: Panel, name: 'listPanel', title: 'Data Grid List', headerComponents: [
+			{kind: ToggleButton, content: 'Selection', name: 'selectionToggle', onChange: 'selectionChanged'},
 			{kind: ContextualPopupDecorator, components: [
 				{kind: ContextualPopupButton, content: 'Selection Type'},
 				{kind: ContextualPopup, classes: 'moon-4h', components: [
-					{kind: RadioItemGroup, name: 'selectionTypeGroup', components: [
+					{kind: RadioItemGroup, name: 'selectionTypeGroup', onActiveChanged: 'selectionTypeChanged', components: [
 						{content: 'Single', value: 'single', selected: true},
 						{content: 'Multiple', value: 'multi'},
 						{content: 'Group', value: 'group'}
 					]}
 				]}
 			]},
+			{kind: ContextualPopupDecorator, components: [
+				{kind: ContextualPopupButton, content:'Item Type'},
+				{kind: ContextualPopup, classes:'moon-6h', components: [
+					{kind: RadioItemGroup, name: 'itemTypeGroup', onActiveChanged: 'itemTypeChanged', components: [
```

Comment:
After some further discussion, we want to fully resolve the issue here before integrating this change. So @germboy @ryanjduffy, please see the first comment when you have a chance, thanks!